tests/integration-test/genesis.json

```diff
@@ -14,9 +14,6 @@
     "londonBlock": 0,
     "archimedesBlock": 0,
     "shanghaiBlock": 0,
-    "bernoulliBlock": 2,
-    "curieBlock": 3,
-    "darwinTime": 1719558098,
```

Comment:
why delete this?

Reply:
to make the diff compared with `develop` branch smaller, I thought this was added by accident.

Reply:
OK.

Reply:
added back in [4645ccf](https://github.com/scroll-tech/scroll/pull/1470/commits/4645ccffae4d62f59d3824db6d1b523a039a0263).